NEWS.md

```diff
@@ -1,4 +1,3 @@
-
 ROhdsiWebApi 1.2.1
 ==================
 
```

Comment:
This sould have a line like "implemented export of cohort definition sets for study packages"